protostar/testing/environments/fuzz_test_execution_environment.py

```diff
@@ -130,7 +135,6 @@ def build_and_run_test(
     ):
         try:
 
-            @multiply_decorator(example, self.state.config.fuzz_examples)
```

Comment:
nitpick

```python
@self.decorate_with_examples
```

should work

but I think this would be even better:

```python
def many_examples(examples):
    def inner(f):
        for ex in reversed(examples):
            f = example(ex)(f)
        return f
    return inner

# here
@many_examples(self.state.config.fuzz_examples)
...
```

no need to test `many_examples` in this scenario, integration tests do the job already

Reply:
```
@self.decorate_with_examples
```
Yes, this works.

As for the other example you provided (with `many_examples`), it doesn't work. For me it looks like there are a lot of ways of doing this, we already managed to have a small amount of code that does a job and I wouldn't spend too much time on this at this point. I think the most beneficial approach would be to just choose one of the possibilities and move on.

Reply:
[use the example decorator as annotation](https://github.com/software-mansion/protostar/pull/947/commits/525b1dc8eeb3377c4700c9dd146a9e354208200a)

Reply:
yep, hence I was nitpicking :)